### DISCUSSION TITLE
Rotational invariance

### DISCUSSION THREAD

**Discussion by RicardoCecconello** [ttadano/alamode#221] [Q&A] [answered]
Dear Dr. Tadano,
The constraints for rotational invariance are applied for improper rotations (rotation + translation) of the space group?

**Comment by ttadano** [comment, marked as answer]
The rotation you mentioned is the rotational part of the space group operation. This operation is always performed in ALM.

The rotational invariance (`ICONST=2`) considers extra constraints between force constants so that the potential energy does not change with an infinitesimal rigid rotation of the system.
